ocaml/xapi/monitor_pvs_proxy.ml

```diff
@@ -60,7 +57,7 @@ let get_changes () =
                 | Rrd.VT_Float v -> int_of_float v
                 | Rrd.VT_Unknown -> -1
               in
-              if ds.Ds.ds_name = Xapi_globs.pvs_proxy_status_ds_name then
+              if ds.Ds.ds_name = "pvscache_status" then
```

Comment:
Why would you "unglobal" this? I think that it is good practice to have constant strings factored out.

Reply:
To open a conversation ;)

It's not a "xapi global" constant string at all, it gives the wrong impression that is shared or reused, at this point having it as a global provides indirection without benefit.

It might make sense to put it into xcp-idl and share the value with the plugin that generates the data. Same with the other metrics-related string constants.